### PR TITLE
fix Issue 16308 - [ndslice] should always has save primitive

### PR DESCRIPTION
**Pull request by 9il**
The reason is that Slice never calls front/popFront and others
primitives of underlaying range.

### CODE REVIEW

File: std/experimental/ndslice/selection.d [hunk 1]
```diff
@@ -1632,12 +1630,6 @@ template IndexSlice(size_t N)
     {
         private size_t[N-1] _lengths;
 
-        IndexMap save() @property const
```
Comment (9il) [inline std/experimental/ndslice/selection.d]: removing this automatically make force the unittest to test the changes

**Comment by JackStouffer** [comment]
Seems like `save` is pretty well tested, so because the tests are green LGTM

**Comment by skoppe** [comment]
👍

**Comment by qznc** [comment]
LGTM :+1:

**Comment by 9il** [comment]
rebased

**Comment by wilzbach** [comment]
:+1:  -  let's get this into master (ndslice is stil experimental, after all)

**Comment by 9il** [comment]
Random free-list error. How can I restart auto-tester?

**Comment by wilzbach** [comment]
> Random free-list error. How can I restart auto-tester?

image

find the failing build & click on deprecate (I just did that for this build).

**Comment by 9il** [comment]
> find the failing build & click on deprecate (I just did that for this build).

Thanks!

**Comment by wilzbach** [comment]
Anyone interested in merging this fix? It has four 👍 and even removes code :)

**Comment by 9il** [comment]
Thanks!

**Pull request merged**